### DISCUSSION TITLE
请问会增加自定义命名的功能吗？

### DISCUSSION THREAD

**Discussion by ENuiOWyTv55o** [yutto-dev/yutto#32] [General]
下载后的命名规则好像是直接使用的视频的标题，一次性下载大量视频后就无法管理视频发布时的先后顺序。
请问参数里有没有新下载文件的自定义命名规则的参数？
后续能否考虑添加 “上传时间年月日+视频标题”的的功能？
感谢！

**Comment by SigureMo** [comment]
上传时间参数嘛？这个后续会考虑一下～不过可能需要先看看有没有相关 API

**Reply by ENuiOWyTv55o** [reply]
感谢！期待后续更新。

**Comment by SigureMo** [comment]
新的上传时间参数 pubdate 已在 #42 中得到支持（2.0.0-bata.8）～可根据需要参考文档对路径进行调整～

比如目前用户全部收藏夹的默认下载路径模板是 `{username}的收藏夹/{fav_title}/{title}/{name}`，如果想在视频名前增加上传时间，只需增加参数 `-tp "{username}的收藏夹/{fav_title}/{title}/{pubdate}-{name}"` 即可～
